example/http/client/Jamfile

```diff
@@ -16,3 +16,5 @@ build-project sync ;
 build-project async-ssl ;
 build-project coro-ssl ;
 build-project sync-ssl ;
+
+build-project body ;
```

Comment:
json-body ?

Reply:
I named it body in case we get more body examples.